packages/user/Symbol/service/src/lib.rs

```diff
@@ -0,0 +1,309 @@
+#[psibase::service_tables]
+pub mod tables {
+    use async_graphql::{ComplexObject, SimpleObject};
+    use psibase::check_none;
+    use psibase::services::nft::{NftRecord, Wrapper as Nft};
+    use psibase::services::tokens::Wrapper as Tokens;
+    use psibase::services::tokens::{Decimal, Quantity};
+    use psibase::{
+        check, check_some, get_sender,
+        services::{nft::NID, tokens::TID},
+        AccountNumber, Fracpack, Table, ToSchema,
+    };
+    use serde::{Deserialize, Serialize};
+
+    #[table(name = "ConfigTable", index = 0)]
+    #[derive(Serialize, Deserialize, ToSchema, Fracpack, Debug)]
+    pub struct Config {
+        pub billing_token: TID,
+    }
+
+    impl Config {
```

Comment:
Perhaps we shouldn't actually allow for the billing token to be changed once set (for now)? I suspect if it ever changed we would need to reset all of the symbol prices to defaults, and it would probably be atomic with changing the system token itself, which we aren't going to fully support for a while yet.